acceptance/build_info_test.go

```diff
@@ -31,6 +31,7 @@ func TestBuildInfo(t *testing.T) {
 	l := localcluster.Create(1, stopper)
 	l.Start()
 	defer l.Stop()
+	defer assertNoEvents(t, l)
```

Comment:
I understand we don't want it every time; I'm just wondering if we could make this the default state which could be disabled from tests that restart nodes, instead of the other way around.